tests/sycl/Makefile

```diff
@@ -15,8 +15,10 @@ hip_omp_cpu: CC = syclcc
 hip_cuda_gpu: CC = syclcc
 #hip_cuda_gpu: CFLAGS += --hipsycl-targets=cuda:sm_80
 
+# this rule is erroring out on CSD3: "clang-15: error: cannot specify -o when generating multiple output files"
 all: oneapi_intel_cpu oneapi_cuda_gpu hip_intel_cpu hip_intel_gpu
 
+# this is erroring out on CSD3: "clang-15: error: cannot specify -o when generating multiple output files"
```

Comment:
I don't get any errors when I use `dpcpp` from `/usr/local/software/intel/oneapi/2022.1/` to compile this. Was the error with the Intel open-source compiler?

Maybe we should add some comments to the main Makefile explaining that it needs the correct modules loaded and won't necessarily work out of the box on all platforms.

Reply:
On CSD3 it is probably safer to use `Makefile.csd3`. For me `make -f Makefile.csd3 all` compiles all the targets on csd3. Although keeping the two (and possible other platform-specific Makefiles) in sync is a bit of a pain.